packages/core/rendering/core-rendering-server-internal/src/bootstrap/bootstrap_renderer.ts

```diff
@@ -83,7 +84,7 @@ export const bootstrapRendererFactory: BootstrapRendererFactory = ({
     }
 
     const themeTag = getThemeTag({
-      themeVersion,
+      name: !themeName || themeName === 'amsterdam' ? 'v8' : themeName,
```

Comment:
Theme tags should follow the `<name><color_mode>` format, but in case of the current Amsterdam theme they're called `v8light` and `v8dark` instead. Let's keep it this way for compatibility.

Reply:
if the themeName UI setting is `amsterdam` we're retuning `'v8'` and if it's `'v8'` we'll also return `'v8'`. Maybe I'm not understanding what this is supposed to do, but seems like maybe the logic should be `!== 'amsterdam'`?

Reply:
> if the themeName UI setting is amsterdam we're retuning 'v8' and if it's 'v8' we'll also return 'v8'

Yeah, that's technically correct. We're not expecting `v8` to be a value of `themeName` but even if it was it's not going to break anything since it will result in a valid theme tag.

This logic exists only to map `amsterdam` to `v8` - we're decoupling theme names from kibana versioning (like `v7` and `v8`), but there's no real value renaming the current theme to `amsterdam` and all places where `v8` could be used.